src/app/views/app-sections/StatusMessages.tsx

```diff
@@ -4,7 +4,10 @@ import { FormattedMessage } from 'react-intl';
 export function statusMessages(queryState: any, actions: any) {
   return queryState && (
   <MessageBar messageBarType={queryState.messageType}
-  isMultiline={false} onDismiss={actions.clearQueryStatus}>
+  isMultiline={false}
```

Comment:
Indent by two spaces  :wink:

Reply:
I think we should add a pre-commit hook that runs prettier no? 🤔

Reply:
Yes, we should.